### ISSUE TITLE
Are polygons supported?

### ISSUE THREAD

**Issue by miloskroulik** [kartena/printlet#8]
I tried to display polygons and it didn't work with server.js, nor with static.js. Other types of geometry work fine.

My static.js for reference:
````
var fs = require('fs'),
    printlet = require('../lib/printlet'),
    tileJson = require('./osm.json');

// Create a printlet instance from a TileJSON
printlet(tileJson)({
  // Specify the image options
  width: 800,
  height: 600,
  zoom: 12,
  lng: 11.95,
  lat: 57.7,
  format: 'png',
  // Add the features to be drawn
  geojson: {
    "type": "FeatureCollection",
    "features": [
      {
      "type": "Feature",
      "properties": {},
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              11.969261169433594,
              57.71941492162019
            ],
            [
              11.957588195800781,
              57.71588512774503
            ],
            [
              11.95638656616211,
              57.70726548342996
            ],
            [
              11.970462799072266,
              57.70630252339771
            ],
            [
              11.982049942016602,
              57.71368456258319
            ],
            [
              11.978273391723633,
              57.71748962217837
            ],
            [
              11.969261169433594,
              57.71941492162019
            ]
          ]
        ]
      },
        "properties": {
          "style": {
            "strokeStyle": "rgba(700, 0, 0, 1)",
            "lineWidth": "10"
          }
        }      
    },

      // Draw a semi-transparent line
      {
        "type": "Feature",
        "geometry": {
          "type": "LineString",
          "coordinates": [[11.95, 57.7], [12, 58]]
        },
        "properties": {
          "style": {
            "strokeStyle": "rgba(200, 0, 0, 0.6)",
            "lineWidth": "5"
          }
        }
      },
      // Draw a point represented by a text marker
      {
        "type": "Feature",
        "geometry": {
          "type": "Point",
          "coordinates": [11.95, 57.7]
        },
        "properties": {
          "style": {
            "font": "32px serif",
            "fillStyle": "rgb(20, 20, 120)",
            "marker": {
              "text": "Hello World"
            }
          }
        }
      }
    ]
  }
}, function(err, data) {
  if (err != null) throw new Error(err + '\n' + err.stack);
  // Get the image data as a stream and write save it to a file
  data.stream.pipe(fs.createWriteStream('image.png'));
});
````
I'm using latest version of Printlet.

**Comment by icetan** [comment]
I had implemented the parsing of GeoJson Polygon features incorrectly. It just treated polygons as linestrings ignoring the rings.

It should be fixed and published as version 0.4.2

Thank you for pointing this out.

**Comment by miloskroulik** [comment]
Thanks, it's working nicely
